src/Tizen.TV.UIControls.Forms/ContentCreator.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using Xamarin.Forms;
+using XForm = Xamarin.Forms;
+
+namespace Tizen.TV.UIControls.Forms
+{
+    internal class ContentCreator
+    {
+        internal static View CreateContent(DataTemplate template, Object data, double heightRequest, double widthRequest)
+        {
+            if (template != null)
+            {
+                var content = template.CreateContent();
+                if (content is View view)
+                    return view;
+                else if (content is ViewCell viewCell)
+                    return viewCell.View;
+                else if (content is ImageCell imageCell)
+                    return CreateContent(imageCell, heightRequest, widthRequest);
+                else if (content is TextCell textCell)
+                    return CreateContent(textCell);
+            }
+            return CreateContent(new TextCell { Text = data?.ToString() });
+        }
+
+        static View CreateContent(ImageCell cell, double heightRequest, double widthRequest)
+        {
+            XForm.Label text = new XForm.Label
+            {
+                HorizontalTextAlignment = TextAlignment.Start,
+                VerticalTextAlignment = TextAlignment.Start,
+                HorizontalOptions = LayoutOptions.FillAndExpand,
+                VerticalOptions = LayoutOptions.FillAndExpand,
+                FontAttributes = FontAttributes.Bold,
+            };
+            XForm.Label detailLabel = new XForm.Label
+            {
+                HorizontalTextAlignment = TextAlignment.Start,
+                VerticalTextAlignment = TextAlignment.Start,
+                HorizontalOptions = LayoutOptions.FillAndExpand,
+                VerticalOptions = LayoutOptions.FillAndExpand,
+            };
+            detailLabel.FontSize = Device.GetNamedSize(NamedSize.Micro, detailLabel);
+
+            text.SetBinding(XForm.Label.TextProperty, new Binding("Text", source: cell));
+            text.SetBinding(XForm.Label.TextColorProperty, new Binding("TextColor", source: cell));
+
+            detailLabel.SetBinding(XForm.Label.TextProperty, new Binding("Detail", source: cell));
+            detailLabel.SetBinding(XForm.Label.TextColorProperty, new Binding("DetailColor", source: cell));
+
+            XForm.Image image = new XForm.Image
+            {
+                HeightRequest = heightRequest,
+                WidthRequest = widthRequest,

```

Comment:
no need to set HeightRequest,WidthRequest, 
because, it was added into absolute layout

Reply:
I will fix it. Thank you.